src/charm.py

```diff
@@ -175,6 +187,29 @@ def _on_orchestrator_available(self, event: OrchestratorAvailableEvent):
             return
         self.unit.status = ActiveStatus()
 
+    def _on_lte_core_relation_joined(self, event: RelationJoinedEvent):
+        """Triggered when lte-core relation is joined.
+
+        AGW will provide the IP address of gtp_br0 interface if that is available.
```

Comment:
We should be providing the `MME` (`eth1`) address not the `gtp_br0`, shouldn't we?

Reply:
Yes, Bart is right. Actually, that's my bad, I've probably explained that fast and said that wrongly to you @saltiyazan.

Reply:
If you want to know how's the `gtp_br0` interface involved in the traffic within the AGW you can read [here](https://docs.magmacore.org/docs/lte/openvswitch#useful-utilities-for-debugging-datapath-issues)